docs/software/home_automation.md

```diff
@@ -221,4 +222,61 @@ Official website: <https://www.openhab.org/>
 Official docs: <https://www.openhab.org/docs/>  
 Official community: <https://community.openhab.org/>
 
+## Homebridge
+
+Homebridge allows you to integrate with smart home devices that do not natively support HomeKit. There are over 2,000 Homebridge plugins supporting thousands of different smart accessories.
+
+![Homebridge logo](../assets/images/dietpi-software-homeautomation-homebridge.svg){: width="300" height="300" loading="lazy"}
+
+=== "Access to the web interface"
+
+    The web interface is accessible via port **8581**:
+
+    - URL: `https://<your.IP>:8581`
+    - Username/Password: Are set at first web interface login
+
+=== "Directories"
+
+    - Install directory: `/usr/local/bin/hb-service`
+    - Config, Log & Backup folder: `/var/lib/homebridge`
+
+=== "View logs"
+
+    ```sh
+    journalctl -u homebridge
+    cat /var/lib/homebridge/homebridge.log
```

Comment:
Might be actually an idea to move all logs to journal. Is the `homebridge.log` actually already a duplicate of what is logged to stdout/journal already? At least the journal contains more than than what systemd itself logs about the service start. Might reduce some redundancy and disk I/O if we disable the plain text log file.

However, until we change something, the info here is of course correct.

Reply:
No, unfortunately the journal has a basic logging like this:
```
root@pi:/opt/homebridge# journalctl -u homebridge
Jul 27 00:37:35 pi systemd[1]: Starting homebridge.service - Homebridge...
Jul 27 00:37:35 pi run-parts[3733]: Ensuring /var/lib/homebridge and /opt/homebridge are owned by homebridge
Jul 27 00:37:35 pi systemd[1]: Started homebridge.service - Homebridge.
Jul 27 00:37:36 pi start.sh[3735]: [7/27/2023, 12:37:36 AM] [HB Supervisor] Homebridge Storage Path: /var/lib/homebridgeJul 27 00:37:36 pi start.sh[3735]: [7/27/2023, 12:37:36 AM] [HB Supervisor] Homebridge Config Path: /var/lib/homebridge/config.json
Jul 27 00:37:36 pi start.sh[3735]: [7/27/2023, 12:37:36 AM] [HB Supervisor] Logging to /var/lib/homebridge/homebridge.log
```

and I don't know a way to disable the logging / move the logging to the journal.

Reply:
I cannot find a way to change this either. The official docs do not give a hint either: https://github.com/homebridge/homebridge/wiki/Homebridge-Config-JSON-Explained

I guess it is some library controlling the logs, hence with hidden config keys for it. However, not that important, we can look into it or ask at the HB community at a later time.